### PR TITLE
Add explicit download path message

### PR DESCRIPTION
**Pull request by vivekiitkgp**
If a the download was complete but setup failed, user might be in fix when asked again for download during next attempt. Clearly state the download path.

Signed-off-by: Vivek Rai <vivekraiiitkgp@gmail.com>

### CODE REVIEW

**Comment by vivekiitkgp** [comment]
An alternative would be to save the config file after each step.

File: bin/sequenceserver [hunk 1]
```diff
@@ -245,6 +245,8 @@ MSG
           puts
           puts 'Installing NCBI BLAST+.'
           puts "RUBY_PLATFORM #{RUBY_PLATFORM}"
+          puts 'Archive will be extracted in ~/.sequenceserver directory.'
```
Comment (yeban) [inline bin/sequenceserver]: @vivekiitkgp How about we instead change the 'installing ncbi blast+' msg (L246) to 'Installing NCBI BLAST+ to ~/.sequenceserver/ncbi-...'?

Comment (vivekiitkgp) [inline bin/sequenceserver]: @yeban It should work as well. Although, I'm not sure how to interpret it if the *installation* fails. Was it installed? We can be even more verbose I guess.

Comment (yeban) [inline bin/sequenceserver]: There's ambiguity in 'Archive will be extracted ...' msg too. Nonetheless, this definitely improves on what we have, so I'm merging the PR.

**Pull request merged**